src/pallene/pallenelib.lua

```diff
@@ -46,6 +46,10 @@ return [==[
 #include <stdbool.h>
 #include <stdlib.h>
 
+/* Pallene Tracer for tracebacks (dynamically linked). */
+/* Look at https://github.com/pallene-lang/pallene-tracer for more info. */
```

Comment:
This repo is still empty, by the way.
Is that intentional?

Reply:
Switch to the `dev` branch. And also please review the PR which intends to merge the `dev` branch to `main`.